last_seen.py

```diff
@@ -0,0 +1,28 @@
+from brutal.core.plugin import BotPlugin, cmd, event
+from time import strftime, gmtime
+
+class LastSeen(BotPlugin):
+    def setup(self, *args, **kwargs):
+        self.seen = self.open_storage('last_seen')
+
+    @event
+    def seen_evt(self, event):
+        events = ['quit', 'part', 'kick', 'join', 'rename', 'topic']
+        if event.event_type in events:
+            nick = event.meta['nick']
+            time = strftime("%d.%m.%Y", gmtime())
+            self.seen[nick] = time
```

Comment:
@JakubNvk personally, I'd also like to know the event (was it a message? Did the user leave the channel?)